lib/shopifyHmac/constructHmacParameter.test.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+
+const { test } = require('tap');
+const constructHmacParameter = require('./constructHmacParameter');
+
+const defaultOptions = {
```

Comment:
do you need these default options in the test file?

Reply:
This is the pattern I copied from the other tests but we really only need the defaults if we want to test for failures due to not including the correct option parameters. The plugin takes care of this in `plugin.js` but the test files don't benefit from the library defaults because of how we test each file individually.